code/datums/flock/flock.dm

```diff
@@ -169,6 +179,21 @@ var/flock_signal_unleashed = FALSE
 					// enemy no longer exists, let's do something about that
 					src.enemies -= name
 
+		if ("stats")
+			var/list/stats = list(
+				"Drones realized: " = src.drones_made,
+				"Bits formed: " = src.bits_made,
+				"Net deaths: " = src.deaths,
```

Comment:
What about this is net?

Reply:
Just total drone + bit deaths

Reply:
then call it total and not net? that's not what net means.